bundles/org.openhab.binding.broadlink/README.md

```diff
@@ -0,0 +1,93 @@
+# Broadlink Binding
+
+This binding supports a range of home networking devices made by (and occasionally OEM licensed from) [Broadlink](https://www.ibroadlink.com/).
+
+## Supported Things
+
+| Thing ID   | Description                                                             |
+|------------|-------------------------------------------------------------------------|
+| a1         | Broadlink A1 multi sensor                                               |
+| mp1        | Broadlink MP1 WiFi Smart Power Strip (4 sockets)                        |
+| mp1_1k3s2u | Broadlink MP1 1K3S2U WiFi Smart Power Strip (3 sockets, 2 USB)          |
+| mp2        | Broadlink MP2 WiFi Smart Power Strip (3 sockets, 3 USB)                 |
+| sp1        | Broadlink SP1 WiFi Smart Socket                                         |
+| sp2        | Broadlink SP2 WiFi Smart Socket with night light                        |
+| sp3        | Broadlink SP3/Mini WiFi Smart Socket with night light                   |
+| sp3s       | Broadlink SP3s WiFi Smart Socket with Power Meter                       |
+| rm         | Broadlink RM WiFI IR Transmitter                                        |
+| rm2        | Broadline RM2/Pro WiFi IR/RF Transmitter with temperature sensor        |
+| rm3        | Broadlink RM3/Mini WiFi IR Transmitter                                  |
+| rm3q       | Broadlink RM3 WiFi IR Transmitter with Firmware v44057                  |
+| rm4        | Broadlink RM4 WiFi IR Transmitter with temperature and humidity sensors |
+
+## Discovery
+
+Devices in the above list that are set up and working in the Broadlink mobile app should be discoverable by initiating a discovery from the OpenHAB UI.
```

Comment:
The correct way to spell it is `openHAB` please change all occurrences to this, be careful if using find and replace that any folder paths do not get changed by mistake.